ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -787,6 +788,18 @@ StackView {
                     Component.onCompleted: root.registerBurnTokenFeesSubscriber(burnTokensFeeSubscriber)
                 }
             }
+
+            TransferOwnershipPopup {
```

Comment:
Second instance detected; what about instead moving to a central place, (`Popups.qml`)?